app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+import { Nav } from 'reactstrap'
+import apts from '../assets/apts.gif'
+
+
+
+class Header extends Component {
+    render() {
+        console.log("logged in:", this.props.logged_in)
+        console.log("user:", this.props.current_user)
+        console.log("sign up:", this.props.new_user_route)
+        console.log("sign in:", this.props.sign_in_route)
+        console.log("sign out:", this.props.sign_out_route)
+        const {logged_in, current_user, new_user_route, sign_in_route, sign_out_route} = this.props
+      return (
+
+       <header>
+        <NavLink to="/">
+         <img src={apts} alt="logo for Apt Finder" className="apts"/>
+        </NavLink>
+        <div className="nav-links">
+      
+          {! logged_in &&
+          <>
+          <ul>
+            <a href="http://localhost:3000/apartmentindex"><div className="link-text">View Apartments</div></a>
```

Comment:
This looks like internal navigation inside the React app. You'll want to use a NavLink here instead of an `a` tag